scripts/plot_radiosonde_data.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Plot radiosonde level 1 data
+
+Created on Sun Jan 29, 2023
+
+@author: laura
+
+run the script:
+    python make_plots_radiosonde.py -i inputfilename -o outputdirectory
+for level 1 radiosonde files. The inputfilename should include the direction (ascent or descent). If not, no SkewT diagram is made.
+
+This script produces 3 plots:
+    - trajectory
+    - temperature T, dew point tau, relative humidity rh, water vapor mixing ratio, wind speed, and wind direction
+    - SkewT diagram (only if direction, i.e. ascent or descent, is given in inputfilename)
+"""
+
+import argparse
+import os.path
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import metpy.calc as mpcalc
+import numpy as np
+import xarray as xr
+from metpy.plots import Hodograph, SkewT
+from metpy.units import units
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i",
+        "--inputfile",
+        metavar="INPUT_FILE",
+        help="Converted sounding file (netCDF)",
+        default=None,
+        required=True,
+    )
+
+    parser.add_argument(
+        "-o",
+        "--outdir",
+        metavar="/some/example/path/",
+        help="Output directory for plots",
+        default="/Users/laura/ownCloud/Campaigns/Radiosonden/plots",
+        required=False,
+    )
+
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        metavar="DEBUG",
+        help="Set the level of verbosity [DEBUG, INFO," " WARNING, ERROR]",
+        required=False,
+        default="INFO",
+    )
+
+    parsed_args = vars(parser.parse_args())
+
+    return parsed_args
+
+
+def main():
+    args = get_args()
+
+    # Define input file
+    filename = args["inputfile"]
+
+    data = xr.open_dataset(filename)
```

Comment:
```suggestion
    data = xr.open_dataset(filename)
    data = data.pint.quantify()
```
This reads in the units given in the metadata of the dataset and makes the variables unit aware.

Reply:
Maybe you need to use
```
data = data.pint.quantify(unit_registry=units)
```
to have the same unit registry as metpy otherwise it might get you some errors.